Chapters/2-2-Chapter9.md

```diff
@@ -0,0 +1,169 @@
+
+He continued to cough uncontrollably, struggling to regain control of his breathing. But then he felt his muscles relaxing, almost to the point of being immovable, and his head was rushing with countless thoughts and feelings; if he stopped to consider any of them, they would slip away in the next moment. His breathing became automatic. A passenger in his own body.
+
+"There you go," Shinna said while thi laid him down on the floor. The soft fabric was warm to the touch. He could feel hae hands traveling across his skin and multiplying to eight, twelve hands. His sense of self began to blur and he wasn't sure where his self ended and Shinna or Setre began. He put his hand around the pendant on his neck.
+
+"I want to leave," he whispered to the air.
+
+"Shh," Shinna said as thi moved his hand away, "We are all children of Ulu."
+
+The floor became an ocean of silk and he was sinking deeper into it with every movement. _"All of your dreams."_ He reached out to touch the blue patterns in his eyes, his mind struggling to regain control of his newly autonomous body.
+
+Thi said, "Never forget... I own you."
+
+His memory of reality faded into the night, hidden forever from the mind where the luckiest of regretful memories live.
+
+In its place, a series of dreams.
+
+_Where am I?_
+
+Kipnu opened his eyes and found himself floating alone on an ocean of thick black fluid. The dark of Ulu lie above him, and a faint halo of light surrounded the horizon. He tried to lift himself up, but the ocean held fast. _Is this the afterlife? Have I died? This feels too real to be a dream._ Slowly, the fluid turned into hundred hands and began to pull him down. He tried fighting against them, but they held on tighter. He held his breath while sinking into the abyss until he couldn't hold it any longer. _Drowning would be ok. Ending would be ok._ His lungs let go, and he let the dark flow in, permeating his blood, until he couldn't feel anything.
+
+He woke up again, but this time he was suspended above Palinan with the full brightness of Ara behind him. Liffa was in a small metal capsule directly in front of him. Thi was screaming trying to tell him something, but he couldn't hear hae.
+
+"Liffa!" he called to hae, but thi couldn't hear him either. He tried to move toward hae, but there was nothing to push against. The heat and light from Ara was suddenly blotted out, and he twisted around to see a giant orb of the black liquid approaching them. It swallowed them both, and Kipnu tried to grab Liffa right as it swallowed him, but thi was lost in the darkness. Kipnu kept swimming through the dark, reaching for the next place Liffa could be, but he was getting nowhere. He lost all feeling before he gave up to the dark again.
+
```

Comment:
Yea, this feels too psychedelic to be an ecstasy-like experience.
While it is true that people react to substances differently, one's experience with a substance should at least be internally consistent. So for Kip, it will help to think about his fears and desires. What his brain chemistry is composed of? How this substance affects those chemicals? It will help you create a more believable (surreal) scene.

Gateway thought exercise for gateway drugs, ANXIETY. Most substances start with reacting to our anxieties, whether they tone down or heighten them. 

So with this in mind, my literary reading of this trance scene is that he fears drowning and is worried about Liffa being taken away forever. Is that correct?

Reply:
damn I hadn't considered that. Probably going to rewrite this whole chapter.